lib/inspec/profile.rb

```diff
@@ -162,7 +168,8 @@ def params
     end
 
     def collect_tests(include_list = @controls)
-      if !@tests_collected
+      unless @tests_collected
```

Comment:
Hrm.  How would we get here, if we're on an unsupported platform?  I thought the code in runner.rb would catch the unsupported platform, issue a warning, and skip adding the profile to the test collector.  Is there another way to get here?

Reply:
I think we need this as the runner only passes in the parent but this will call itself for each dependent.